tests/model_hub_tests/transformation_tests/models/transformations-models-precommit

```diff
@@ -0,0 +1,27 @@
+hf-internal-testing/tiny-random-LlamaForCausalLM,https://huggingface.co/trl-internal-testing/tiny-random-LlamaForCausalLM,,,ts_name:ov::pass::RoPEFusion,layer:RoPE
```

Comment:
`ov::pass::RoPEFusion` is a name of GraphRewrite pass, which includes a set of Matchers: "RoPEFusionGPTNEOX", "RoPEFusionGPTJ" etc . Maybe it's better to be more specific here and check Matcher passes instead of GraphRewrite

Reply:
discussed offline, we don't have logs in GraphRewrite yet